### PR TITLE
Adding CSS Handle to ShowMore button on ReviewText

### PR DESCRIPTION
**Pull request by PedroLuccaSantiago**
#### What problem is this solving?

The ShowMore component, used inside the reviewCommentMessage, does not have a class to customize the element in CSS. This request will add a CSS Handle to this component, allowing it to have a class.

#### How does this PR make you feel? [:link:](https://media1.giphy.com/media/9o59Pga7BWlDrzWhhh/giphy.gif?cid=ecf05e47v997h1d8ttjodpnyr23y31x8ll8dow8t65fi6m6p&rid=giphy.gif&ct=g)

### CODE REVIEW

File: react/Reviews.tsx [hunk 1]
```diff
@@ -860,6 +862,7 @@ function Reviews() {
                           more="Show more"
                           less="Show less"
                           anchorClass=""
+                          className={`${handles.showMoreButton}`}
```
Comment (polishq) [inline react/Reviews.tsx:865]: The `ShowMore` component does not accept a `className` prop. This is currently causing the app build to fail. I believe you should use the `anchorClass` prop instead. Please verify your changes by linking the app in a workspace.

**Review by polishq — CHANGES_REQUESTED** [review]

File: react/Reviews.tsx (hunk 1 above)
Comment (PedroLuccaSantiago) [reply react/Reviews.tsx:865]: Thank's for your review. I will fix this code and try to build the app again. @polishq Thank you!

**Review by polishq — CHANGES_REQUESTED** [review]

File: react/Reviews.tsx [hunk 2]
```diff
@@ -861,8 +860,7 @@ function Reviews() {
                           lines={3}
                           more="Show more"
                           less="Show less"
-                          anchorClass=""
-                          className={`${handles.showMoreButton}`}
+                          anchorClass="showMoreButton"
```
[the remark this answers is not in the document]
Comment (PedroLuccaSantiago) [reply react/Reviews.tsx]: Thanks for the correction! I fixed the addition of the class.

**Review by polishq — APPROVED** [review]
LGTM

**Review by georgebrindeiro — APPROVED** [review]
LGTM

**Pull request merged**